src/Makefile.am

```diff
@@ -26,7 +26,7 @@ BITCOIN_INCLUDES += $(UNIVALUE_CFLAGS)
 
 LIBBITCOIN_SERVER=libbitcoin_server.a
 LIBBITCOIN_COMMON=libbitcoin_common.a
-LIBBITCOIN_CONSENSUS=libbitcoin_consensus.a
+LIBBITCOIN_CONSENSUS=libelements_consensus.a
```

Comment:
I mentioned elsewhere that I think we may actually want to _reduce_ the number of non-user-visible places that we rename things from bitcoin to elements in the build stuff. I guess the consensus library is in theory a build product that is user-visible, which is why to rename that, but not the stuff above and below it?

Making it easier to understand what's going on is not a bad goal, but it might be in tension here with reducing the total amount of divergence with upstream.

Reply:
(For the sake of getting this in, fine to leave it if you prefer.)